parsers/US_PJM.py

```diff
@@ -167,7 +176,7 @@ def fetch_production(
                     production[mode] = row.get("mw")
             data_point = {
                 "zoneKey": zone_key,
-                "datetime": arrow.get(dt).replace(tzinfo="US/Eastern").datetime,
+                "datetime": dt.to_pydatetime().replace(tzinfo=TIMEZONE),
```

Comment:
note: "US/Eastern" has been [renamed](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) to "America/New_York"